js/arena.js

```diff
@@ -0,0 +1,48 @@
+export default class Arena {
+  constructor(name, gladiators = []) {
+    this.name = name.charAt(0).toUpperCase() + name.slice(1);
```

Comment:
should be capitalized meaning, 

> Capitalization is the writing of a word with its first letter in uppercase and the remaining letters in lowercase.

the first should be in upper case, while the rest should be in lowercase